### PR TITLE
Bruce/6 adding footer config

### PR DESCRIPTION
**Pull request by BruceRodrigues**
Closes https://github.com/clevercanary/anvil-portal/issues/6

Adding footer config

### CODE REVIEW

File: next/site-config/anvil-portal/footer.ts [hunk 1]
```diff
@@ -1,9 +1,70 @@
...
+    {
+      label: "Help",
+      url: `${BROWSER_URL}/help`,
+    },
+    {
+      label: "Privacy",
+      url: `${BROWSER_URL}/privacy`,
+    },
+  ],
+  socials: [
```
Comment (frano-m) [inline next/site-config/anvil-portal/footer.ts:46]: Now that we have this in twice, let's make a `socials` constant in the `constants.ts` and reference this for both the `header` and `footer` configs.

**Review by frano-m — CHANGES_REQUESTED** [review]
Hey @BruceRodrigues nice work ⭐ .

Just a few tiny changes and then this one is good to go!

File: next/site-config/anvil-portal/footer.ts [hunk 2]
```diff
@@ -1,9 +1,70 @@
 import { FooterProps } from "@clevercanary/data-explorer-ui/lib/components/Layout/components/Footer/footer";
+import { SOCIAL } from "@clevercanary/data-explorer-ui/lib/components/common/Socials/socials";
+import logoHhs from "../../images/logoHhs.svg";
```
Comment (frano-m) [inline next/site-config/anvil-portal/footer.ts:3]: We can now import images like this `import logoHhs from "images/logoHhs.svg";`

Comment (BruceRodrigues) [reply next/site-config/anvil-portal/footer.ts]: Thanks for fixing this

**Comment by BruceRodrigues** [comment]
Hey @frano-m ! Thanks for the review! This is ready for another look

**Review by frano-m — APPROVED** [review]
Awesome @BruceRodrigues ⭐

**Pull request merged**